Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,65 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        // GUI model loading
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
```

Comment:
This begs  the question why are you not invoking the Add New call back method? Seems that the test could become stale if the method in the GUI changes and the test isn't updated. Seems less error prone to invoke the method that makes these API and Context calls.

Reply:
I agree that it's ideal if the test mimics what the GUI does in a maintainable way, and that moving code into the OpenSimContext could be a way to do that. But I don't know all the details, and in the end, it seems that it's up to @aymanhab to make sure this test remains useful for him. Perhaps the GUI code could have a comment saying "If you update this block of code, update the respective code in TestEditMarkers.java in opensim-core."